third_party/phantomjs/phantomjs.sh

```diff
@@ -19,11 +19,14 @@
 #         export FC_DEBUG="1024"
 
 if [[ "${RUNFILES}" == "" ]]; then
-  if [[ "$(basename "$(pwd)")" =~ \.runfiles$ ]]; then
-    RUNFILES="$(pwd)"
+  if [[ "$(pwd)" =~ ^(.*\.runfiles)(/.*)?$ ]]; then
+    RUNFILES="${BASH_REMATCH[1]}"
   else
     RUNFILES="${0}.runfiles"
   fi
+  if [ -d "${RUNFILES}/io_bazel_rules_closure" ]; then
```

Comment:
1. Not for released version (0.2.1) who has the bug.
2. for now io_bazel_rules_closure/external/phantomjs/, in the future simply phantomjs/ (with a transition periode where both are correct).
3. No and not planed in the near future. We do not do transitive dependencies fetching for many reason, mostly prevent dependency hell.